citylearn/building.py

```diff
@@ -947,7 +1005,8 @@ def get_periodic_observation_metadata() -> Mapping[str, int]:
     def apply_actions(self,
         cooling_device_action: float = None, heating_device_action: float = None,
         cooling_storage_action: float = None, heating_storage_action: float = None, 
-        dhw_storage_action: float = None, electrical_storage_action: float = None
+        dhw_storage_action: float = None, electrical_storage_action: float = None,
+        **kwargs
```

Comment:
Need to review this when we meet; `**kwargs` here is a bit too implicit I think. Better to have a parameter for EVs that is a `dict` and formatted correctly by `citylearn.citylearn.CityLearnEnv._parse_actions`